tests/conftest.py

```diff
@@ -363,3 +367,15 @@ def artifact_data():
     datacatalog = DataCatalog()
     datacatalog.from_predefined_catalogs("artifact_data")
     return datacatalog
+
+
+@pytest.fixture()
+def mock_model(tmpdir):
```

Comment:
Try using `def mock_model(tmpdir, mocker: MockerFixture):`, so that you can use the `pytest-mock` wrapper for making your mocks.